src/main/java/xyz/nucleoid/bedwars/game/active/BwItemGenerator.java

```diff
@@ -107,11 +106,7 @@ private Text getTimerText(long time) {
                 numberFormatting = Formatting.AQUA;
             }
         }
-
-        MutableText titleText = new LiteralText("Next spawn in: ");
-        MutableText numberText = new LiteralText(String.format("%02d:%02d", minutes, seconds));
-
-        return titleText.formatted(titleFormatting).append(numberText.formatted(numberFormatting));
+        return new TranslatableText("floating_text.bedwars.spawn_cooldown", new LiteralText(String.format("%02d:%02d", minutes, seconds)).formatted(numberFormatting)).formatted(Formatting.GOLD);
```

Comment:
Should be formatted gold, like the previous title

Reply:
It is formatted gold